tests/openedx_tagging/core/tagging/test_views.py

```diff
@@ -0,0 +1,266 @@
+        ):
+            assert self.user.has_perm("oel_tagging.add_taxonomy")
+        assert not self.user.has_perm("oel_tagging.add_taxonomy")
+
+    def test_list_taxonomy_queryparams_none(self):
+        url = reverse("taxonomy-list")
+
+        self.client.force_authenticate(user=self.user)
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    @ddt.data(1, 0, True, False, "True", "False", "1", "0")
+    def test_list_taxonomy_queryparams_ok(self, enabled_valid):
+        url = reverse("taxonomy-list")
+
+        self.client.force_authenticate(user=self.user)
+        response = self.client.get(url, {"enabled": enabled_valid})
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    @ddt.data(2, "invalid")
+    def test_list_taxonomy_queryparams_error(self, enabled_invalid):
+        url = reverse("taxonomy-list")
+
+        self.client.force_authenticate(user=self.user)
+        response = self.client.get(url, {"enabled": enabled_invalid})
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_list_taxonomy_permission_anon(self):
+        url = reverse("taxonomy-list")
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_list_taxonomy_permission_user(self):
+        url = reverse("taxonomy-list")
+
+        self.client.force_authenticate(user=self.user)  # ToDo: Create list permission?
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_detail_taxonomy_permission_anon(self):
+        url = reverse("taxonomy-detail", kwargs={"id": self.taxonomy.id})
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_detail_taxonomy_enabled_permission_user(self):
+        url = reverse("taxonomy-detail", kwargs={"id": self.taxonomy.id})
+
+        self.client.force_authenticate(user=self.user)
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_detail_taxonomy_disabled_permission_user(self):
+        url = reverse("taxonomy-detail", kwargs={"id": self.taxonomy_disabled.id})
+
+        self.client.force_authenticate(user=self.user)
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_detail_taxonomy(self):
+        # Happy path
+        url = reverse("taxonomy-detail", kwargs={"id": 123})
+
+        with patch.object(
+            self.user, "has_perm", _has_perm_view_taxonomy
+        ):
+            self.client.force_authenticate(user=self.user)
+            response = self.client.get(url)
+            self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_detail_taxonomy(self):
```

Comment:
Copy-paste error: Because this method is named `test_detail_taxonomy`, the same name as the test above, only the second definition of this method will be run.

I tried to figure out how to enable test coverage on the `tests/*` files too -- thought I just needed to add `tests` to the list of `source` dirs in [`.coveragerc`](https://github.com/open-craft/openedx-learning/blob/rpenido/fal-3449-taxonomy-view-management-apis/.coveragerc#L4-L6), but that didn't work for some reason. Maybe you can figure out why not?

Reply:
Good diagnosis, but no, not yet -- I'd like to keep `.coveragerc` in place so I can run `make quality` locally.
(I can't run `tox` locally because of the mysql testing stuff, which is critical for us before merging, but will take some work to update.)